frontend/src/theme/singleDatePickerStyles.tsx

```diff
@@ -0,0 +1,44 @@
+import colors from "./colors";
```

Comment:
woah you added styling for the date picker 😄 it wasn't part of the PR but it's definitely helpful to have for the future. Unfortunately, we don't have designs for a single date picker so I'll add that to the to-do list for the designers

Reply:
i just changed the colors so they would be purple and removed the ugly blue outline that appears when you click on it :D